src/internal/analytics/components/analytics-funnel.tsx

```diff
@@ -279,16 +279,25 @@ function useStepChangeListener(stepNumber: number, handler: (stepConfiguration:
     };
   }, []);
 
+  useEffect(() => {
+    const handle = setTimeout(
+      () => subStepConfiguration.current.set(stepNumber, getSubStepConfiguration()),
```

Comment:
aren't we making this update inside `stepChangeCallback` already?

Reply:
The value needs to be initialized on the first render too.

Previously it worked because the line was before `listenForSubStepChanges` condition.

Now it is separate code like it should have been in the first place